src/values.rs

```diff
@@ -645,21 +645,6 @@ where
     ///
     /// This function panics in dev builds if this is not the case.
     pub fn new(points: Vec<CurvePoint<T>>) -> Self {
```

Comment:
I think the docs here need to be updated to reflect the change.

Reply:
Done